File: Python/eva02/ec-primer-grado.py
a = int(input("Ingrese el valor de a: "))
b = int(input("Ingrese el valor de b: "))

if a != 0 and -b/a == abs(1):
    x = -b / a
    print(x)
else:
    if b == 0: 
        print("La ecuación tiene infinitas soluciones")
    else:
        print("La ecuación no tiene solución")

File: C/EntradaSalida/ecPrimerGrado.c
#include <stdio.h>

int main() {
    int a, b;
    float x; 

    scanf("%d", &a);
    scanf("%d", &b);

    if (a == 0) {
        if (b == 0) {
            printf("La ecuación tiene infinitas soluciones\n");
        } else {
            printf("La ecuación tiene no tiene solución\n");
        }
    } else {
        x = -b / a;
        printf("%d", x);
    }
    
    return 0;
}